analyses/molecular-subtyping-EPN/02_ependymoma_generate_all_data.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+# Author Teja Koganti (D3B)
+
+import sys
+
+import argparse
+import pandas as pd
+import numpy as np 
+import zipfile
+import statistics
+import  scipy
+from scipy import stats
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--notebook', required = True,
+                    help = 'path to the notebook file')
+parser.add_argument('--gistic', required = True,
+                    help = "gistic zip file")
+parser.add_argument('--subfile-gistic', required = True,
+                    help = "subfile of the zip folder that contains broad values by arm")
+parser.add_argument('--gsva', required = True,
+                    help = "gsva scores file")
+parser.add_argument('--expression', required = True,
+                    help = "expression subset file")
+parser.add_argument('--fusion', required = True,
+                    help = "fusion results file")
+parser.add_argument('--breakpoints', required = True,
+                    help = "breakpoint summary file")
+parser.add_argument('-o', '--outfile', required = True,
+                    help = "output file")
+args = parser.parse_args()
+
+# File to write all the data in 
+outfile = open(args.outfile, "w")
+
+# Reading GISTIC broad_values file for CNA 
+zip=zipfile.ZipFile(args.gistic)
+CNA=pd.read_csv(zip.open(args.subfile_gistic), sep="\t")
+CNA = CNA.set_index("Chromosome Arm")
+
+
+# Reading in gene set enrichment analyses file for GSEA scores for NKKB pathway
+gsva = pd.read_csv(args.gsva, sep="\t")
+gsva_NFKB = gsva.loc[gsva['hallmark_name'] == "HALLMARK_TNFA_SIGNALING_VIA_NFKB"]
+gsva_NFKB = gsva_NFKB.set_index("Kids_First_Biospecimen_ID")
+
+# Reading subset gene expression file
+fpkm_df = pd.read_csv(args.expression, sep = "\t")
+fpkm_df = fpkm_df.set_index("GENE")
+zscore_fpkm_df = fpkm_df.apply(scipy.stats.zscore)
```

Comment:
Since `stats` was imported, lets use it directly.
```suggestion
zscore_fpkm_df = fpkm_df.apply(stats.zscore)
```